src/platform/forms/sass/_m-schemaform.scss

```diff
@@ -266,6 +266,23 @@ fieldset.schemaform-field-template {
   margin-left: 27px;
 }
 
+.schemaform-radio-secondary-text {
+  input[type="radio"]+label::before {
+    margin-top: 0.45em;
+  }
+
+  label {
```

Comment:
I think you could get rid of most of the styles under here by using the utility classes: https://design.va.gov/utilities/